dev_tools/spec/lib/solidus/release_drafter/builder_spec.rb

```diff
@@ -8,54 +8,55 @@
   let(:backend) { 'Solidus Backend' }
   let(:api) { 'Solidus API' }
   let(:categories) { [core, backend, api] }
+  let(:number_link_builder) { ->(number) { "https://github.com/solidusio/solidus/pull/#{number}" } }
 
   describe '#initialize' do
     context 'for an existing draft' do
       it "raises when categories doesn't match" do
         content = <<~TXT.chomp
           ## Solidus Core
-          - Old entry 1 #9 (@bob)
+          * Old entry 1 by @bob in https://github.com/solidusio/solidus/pull/9
```

Comment:
@elia, as a nice side effect, we got rid of the [helper method](https://github.com/solidusio/solidus/pull/4908#discussion_r1097628573) 🙂